app/Schema/bank_account_schema.py

```diff
@@ -1,25 +1,25 @@
 from app import ma
 from marshmallow.validate import Length, Range, Regexp
-from marshmallow import fields, INCLUDE
+from marshmallow import fields
 from random import randint
 
 name_string = '^[a-zA-Z]*$'
+address_string = '^[a-zA-Z ]*$'
 
 
 class BankAccountSchema(ma.Schema):
     """Adding the Schema validations to the BankAccount """
-    user_id = fields.Int(required=True)
-    account_type_id = fields.Int(required=True)
-    branch_id = fields.Int(required=True)
-    balance = fields.Int(required=True, validate=Range(min=1000, max=50000, error="Amount greater than 1000"))
+    user_id = fields.Int(strict=True, required=True)
```

Comment:
What is difference between `strict` and `required`?

Reply:
required=True is because if we do not entered the any field or forgot to enter it ,it gives the error as "Missing the required field" ,And strict =True is taken for integer fields that strictly sticks to the integer and doesn't allow the user the pass the integer in string format.